roxie/udplib/udptrs.cpp

```diff
@@ -36,7 +36,7 @@
 
 unsigned udpOutQsPriority = 0;
 unsigned udpMaxRetryTimedoutReqs = 0; // 0 means off (keep retrying forever)
-unsigned udpRequestToSendTimeout = 5; // value in sec
+unsigned udpRequestToSendTimeout = 5000; // value in milliseconds
```

Comment:
Same thing - I suppose for backward compatibility ?
Platform upgrades do not always overwrite/append to existing environment.xml
so to get this new setting I think we should make default 0 here and in getPropInt() above ?